src/runtime_src/core/pcie/driver/linux/xocl/subdev/ospi_versal.c

```diff
@@ -64,6 +64,8 @@
 #include "../xocl_drv.h"
 #include "xrt_drv.h"
 
+// #define OSPI_VERBOSE
```

Comment:
I think it is better not to use "c99" comments style in kernel code

Reply:
removed